tests/unit/client.py

```diff
@@ -106,6 +111,7 @@ def make_config():
 
         ])
         f.write(config)
+        f.close()
```

Comment:
@SahilTikale, I think you're looking at an old diff; I don't see the statement on @henn's most recent commit.